demos-ssi/gc-intranet/content-en.shtm

```diff
@@ -6,113 +6,120 @@
 <head>
 <meta charset="utf-8" />
 <!--#set var="pg-title-en" value="Content page - GC Intranet theme - SSI variant - Web Experience Toolkit"-->
+<!--#set var="pg-description-en" value=""-->
 <!--#set var="pg-creator-en" value=""-->
 <!--#set var="pg-subject-en" value=""-->
 <!--#set var="pg-issued" value="2011-10-31"-->
-<!--#set var="pg-modified" value="YYYY-MM-DD"-->
+<!--#set var="pg-modified" value="2015-08-11"-->
 <!--#include virtual="/dist-ssi/theme-gc-intranet/common/css-scripts.shtm"-->
 </head>
 <body vocab="http://schema.org/" typeof="WebPage">
 <!--#include virtual="/dist-ssi/theme-gc-intranet/common/header.shtm"-->
 <section>
-	<h2>Heading 2 (<code>h2</code>) - default appearance</h2>
+	<h2>Heading&#32;2&#32;(<code>h2</code>)</h2>
```

Comment:
@nschonni the filler was grab from the actual WET filler https://github.com/wet-boew/wet-boew/blob/master/site/includes/placeholdercontent.hbs.